s/ecr-utils.sh

```diff
@@ -13,6 +13,7 @@ ecr-prepare-action() {
   local var_token="$1"
   shift
   $VERBOSE && print_current_profile_info
+  [ -z "$(kube.utils.has_api_server)" ] && _FATAL -s 1 "Unable to connect to the Entando platform" 
```

Comment:
This is ok for ecr but I would investigate if it can be useful in other top level commands. For example:
```
cecchisandrone@MBPdiAlessandro hackaton % ent app-info
➤ [I] | 2022-12-16 15-58-52 | NODES:
The connection to the server quickstart-entando.awsentando.net:6443 was refused - did you specify the right host or port?
➤ [I] | 2022-12-16 15-58-52 | PODS:
The connection to the server quickstart-entando.awsentando.net:6443 was refused - did you specify the right host or port?
➤ [I] | 2022-12-16 15-58-52 | Application Ingress path:
The connection to the server quickstart-entando.awsentando.net:6443 was refused - did you specify the right host or port?
➤ [E] | 2022-12-16 15-58-53 | Unable to determine the app-builder ingress url
```

Reply:
I have talked about it with @w-caffiero-entando, for this ticket I added the check in ent app-info for application ingress path and in ent prj deploy. They are related to the reachability of the app.
For reachability of cluster, I opened another ticket.